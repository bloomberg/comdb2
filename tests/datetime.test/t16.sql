SET TIMEZONE UTC
CREATE TABLE lhs (i INTEGER)$$
CREATE TABLE rhs (i INTEGER, dt DATETIME)$$
INSERT INTO lhs VALUES (1), (2), (3)
INSERT INTO rhs VALUES (2, '2021-03-05')
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
SELECT CASE WHEN dt IS NULL THEN '1970-01-01' ELSE dt END AS dt FROM lhs LEFT JOIN rhs ON lhs.i = rhs.i ORDER BY dt DESC
