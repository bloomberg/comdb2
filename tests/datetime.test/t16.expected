(rows inserted=3)
(rows inserted=1)
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="2021-03-05T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
(dt="1970-01-01T000000.000 UTC")
